frontend/src/__tests__/cypress/cypress/e2e/distributedWorkloads/GlobalDistributedWorkloads.cy.ts

```diff
@@ -71,5 +103,52 @@ describe('Workload Metrics', () => {
     globalDistributedWorkloads.findHeaderText().should('exist');
   });
 
-  // TODO mturley other tests here for tab navigation, empty states, etc.
+  it('Defaults to Project Metrics tab and automatically selects a project', () => {
+    initIntercepts({});
+    globalDistributedWorkloads.visit();
+
+    cy.url().should('include', '/projectMetrics/test-project');
+    // TODO mturley replace this with real identifiable text on the loaded tab when it is completed
+    cy.findByText('TODO tab content for project metrics -- these are placeholders').should('exist');
+  });
+
+  it('Tabs navigate to corresponding routes and render their contents', () => {
+    initIntercepts({});
+    globalDistributedWorkloads.visit();
+
+    cy.findByLabelText('Workload status tab').click();
+    cy.url().should('include', '/workloadStatus/test-project');
+    cy.findByText('Status overview').should('exist');
+
+    cy.findByLabelText('Project metrics tab').click();
+    cy.url().should('include', '/projectMetrics/test-project');
+    // TODO mturley replace this with real identifiable text on the loaded tab when it is completed
+    cy.findByText('TODO tab content for project metrics -- these are placeholders').should('exist');
+  });
+
+  it('Changing the project and navigating between tabs or to the root of the page retains the new project', () => {
+    initIntercepts({});
+    globalDistributedWorkloads.visit();
+    cy.url().should('include', '/projectMetrics/test-project');
+
+    cy.findByLabelText('Project: Test Project').click();
+    cy.findAllByRole('menuitem').contains('Test Project 2').click();
```

Comment:
Does `cy.findDropdownItem` work here?

Reply:
Good call, I hadn't noticed the usage of that elsewhere. Pushed a change to duplicate what's done in Pipelines (`selectProjectByName` method which uses `findDropdownItem`).